src/p5.dance.js

```diff
@@ -895,19 +892,23 @@ module.exports = class DanceParty {
     return this.p5_.allSprites.indexOf(sprite) > -1;
   }
 
+  /**
+   * @return {Object} TODO: describe
```

Comment:
Non-blocking, but worth coming back to. The documentation is improving the readability so much - thanks!